src/lib/drupal/query.ts

```diff
@@ -125,6 +125,9 @@ export async function fetchSingleEntityOrPreview(opts, type, params) {
 
 // Helper function to return a consistent set of base fields for resources.
 export const entityBaseFields = (entity: NodeTypes): PublishedEntity => {
```

Comment:
```suggestion
export const entityBaseFields = (entity: NodeTypes): PublishedEntity | null => {
```

This makes sense, I think, but when we go to enable strict mode, I expect this will cause a lot of ruckus. 😅